apps/condo/domains/billing/components/ServicesModal.tsx

```diff
@@ -113,6 +114,7 @@ export const ServicesModal: React.FC<IServicesModalProps> = ({
             <SubText size={configSize}>
                 {address}{unitName ? `, ${UnitTypePrefix}. ${unitName}` : ''}
             </SubText>
+            <SubText size={configSize}>{fullName}</SubText>
```

Comment:
What if there are no fullname? Will there be a new blank space?

Reply:
good catch, fixed it